src/index.html

```diff
@@ -5,9 +5,315 @@
+
+        <article class="gallery__article">
+          <div
+            class="gallery__picture-wrapper gallery__picture-wrapper--ratio--2-1">
+            <img
+              class="gallery__picture"
+              src="images/photos/1.jpg"
+              alt="marina palms">
+            <img
+              class="gallery__picture"
+              src="images/photos/2.jpg"
+              alt="marina palms">
+          </div>
+          <h3 class="gallery__article-title">
+            <span class="gallery__blue-text">Marina Palms</span>
+            / North Miami Beach, FL 33162
+          </h3>
+        </article>
+        <article class="gallery__article">
+          <div
+            class="gallery__picture-wrapper gallery__picture-wrapper--ratio--1-1">
+            <img
+              class="gallery__picture"
+              src="images/photos/3.jpg"
+              alt="Edition Residens">
+            <img
+              class="gallery__picture"
+              src="images/photos/4.jpg"
+              alt="Edition Residens">
+          </div>
+          <h3 class="gallery__article-title">
+            <span class="gallery__blue-text">Edition Residens</span>
+            / Miami Beach, FL 33139
+          </h3>
+        </article>
+        <article class="gallery__article">
+          <div
+            class="gallery__picture-wrapper gallery__picture-wrapper--ratio--1-2">
+            <img
+              class="gallery__picture"
+              src="images/photos/5.jpg"
+              alt="Faena House">
+            <img
+              class="gallery__picture"
+              src="images/photos/6.jpg"
+              alt="Faena House">
+          </div>
+          <h3 class="gallery__article-title">
+            <span class="gallery__blue-text">Faena House</span>
+            / Miami Beach, FL 33140
+          </h3>
+        </article>
+
+        <div class="gallery__btn-wrapper">
+          <div class="button">
+            <button
+              class="button__link button__link--color--white"
+              type="button"
+            >
+              View all
+            </button>
+          </div>
+        </div>
+
+      </section>
+      <!--</editor-fold>-->
+
+      <!--<editor-fold desc="Testimonials">-->
+      <section class="reviews" id="reviews">
+        <h2 class="section-title reviews__title">Testimonials</h2>
+
+        <div class="reviews__review-wrapper">
+          <article class="reviews__review">
+            <img
+              class="reviews__foto"
+              src="images/people/1.jpg"
+              alt="leslis`s foto"
+            >
+            <h3 class="reviews__name">Leslie Mckinney</h3>
+            <h4 class="reviews__position">CEO</h4>
+            <p class="reviews__text">
+              Nice work, Certificates National Association of Realtors (USA)
+            </p>
+          </article>
+
+          <article class="reviews__review">
+            <img
+              class="reviews__foto"
```

Comment:
```suggestion
              class="reviews__photo"
```

Reply:
london is the capital of great britain :))